app/code/Bulbulatory/Recommendations/etc/config.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+    <default>
+        <recommendations>
+            <general>
+                <enable>1</enable>

```

Comment:
It is better if feature is disabled by default (client decide when turn it on).

Reply:
@rafdol 

> It is better if feature is disabled by default (client decide when turn it on).

I made "disabled" by default!